core/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalDateHistogramTests.java

```diff
@@ -55,7 +55,7 @@ protected InternalDateHistogram createTestInstance(String name,
                                                        List<PipelineAggregator> pipelineAggregators,
                                                        Map<String, Object> metaData,
                                                        InternalAggregations aggregations) {
-        int nbBuckets = randomInt(10);
+        int nbBuckets = randomNumberOfBuckets();
```

Comment:
I meant in general, but I leave that to you. I think for testing that is still a large enough number. But I leave this to you.